app/(main)/markets/AiChat.tsx

```diff
@@ -53,6 +53,15 @@ const fetchMessages = async (id: Address) => {
   return response.json();
 };
 
+function isJSON(message: string) {
+  try {
+    JSON.parse(message);
+  } catch (e) {
+    return false;
+  }
+  return true;
+}
+
```

Comment:
maybe move this to utils

Reply:
Since it's only being used here, I would leave it here. I find it risky to export things only used once, if we don't need this code anymore it's not evident it's not being used anymore since it's being exported.